src/Multilingual-TextConversion/Character.extension.st

```diff
@@ -4,7 +4,7 @@ Extension { #name : #Character }
 Character >> macRomanToUnicode [
 	"Convert the receiver from MacRoman Unicode."
 
-	^MacRomanTextConverter new unicodeToByte: self
+	MacRomanTextConverter new unicodeToBytes: self do: [ :encodedChar | ^encodedChar ]
```

Comment:
why did you use a return statement , '^encodedChar' , in lambda function use to iterate on collection ?

Reply:
can you change the selector because in unicodeToBytes: do: .
do: is very strange in the implementation you only iterate on a collection in one case.

Reply:
Well, if Ombu is broken in pharo8 and can’t correctly encode methods containing non-ascii as utf8, so they fail to read from disk, that’s hardly the fault of the test itself, is it?

I use a return in the block because do: return self. For macroman with no line end convension installed, it’s always 1 - 1, so returning the first element iterated is correct.

 We iterate over all bytes the Unicode maps to in all the cases, hence do:.
It’s just in two out of three cases, “all bytes” will be a single one.